src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilationBuilder.cs

```diff
@@ -345,5 +347,21 @@ public override ICompilation ToCompilation()
                 _customPESectionAlignment,
                 _verifyTypeAndFieldLayout);
         }
+
+        internal static bool IsReadyToRunPlatformNeutral(PEReader peReader)
```

Comment:
https://github.com/dotnet/runtime/blob/34545d790e0f92be34b13f0d41b7df93f04bbe02/src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/PEReaderExtensions.cs#L101 may be a better home for this method

Reply:
I added it there first but it doesn’t have Internal.Runtime so I put it in the builder.

Reply:
Is it just missing `using`s at the top of the file? It should be fine to add them there. This file has R2R-specific methods currently, so adding one more R2R-specific method to it does not look problematic to me.

Reply:
It's a shared code, the error was coming from other project not this one. I can #ifdef it for READYTORUN.